bluemira/builders/coil_supports.py

```diff
@@ -283,7 +284,13 @@ def build_xyz(
         # Finally, make the floor block
         shape_list.append(self._make_floor_block(floatify(v1.x), floatify(v4.x)))
         shape = boolean_fuse(shape_list)
-        component = PhysicalComponent("ITER-like gravity support", shape)
+        component = PhysicalComponent(
+            "ITER-like gravity support",
+            shape,
+            material=get_cached_material(
+                self.build_config.get("material", {}).get("GS")
```

Comment:
Thoughts welcome:
Possibly build_config['material'] should be autocreated in the reactor config and then the materials entry could be a default dict https://docs.python.org/3/library/collections.html#collections.defaultdict. It might mean we can chain gets. Also could well have unintended side effects...probably a bad idea.